ui/icons/hicolor/scalable/actions/presentation-mode.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
As there is no about comment: You checked the license of the icon or created it by yourself?

Reply:
I created a new logo, but it's really nothing saying and I would request someone to create a new one.

Reply:
The question was only about the license...

Reply:
I would just say CC, I will push sth soon.